### PR TITLE
Refinements to Creator CPP docs

### PR DESCRIPTION
**Pull request by slackmoehrle**

### CODE REVIEW

**Comment by slackmoehrle** [comment]
@drelaptop fixed now.

File: en/editors_and_tools/creator_to_cocos2dx.md [hunk 1]
```diff
@@ -21,27 +22,22 @@ Adding C++ and Lua language support to __Cocos Creator__ is easy:
...
+* select __Project__ -> __LuaCPP Support__ -> __Setup Target Project__. It is required to tell __Cocos Creator__ where to build all the necessary files.
 
-  It is required to tell __Cocos Creator__ where to build all the necessary files.
+    ![](creator_to_cocos2dx-img/dialog_options.png "dialog options")
 
 * select __Build__.
 
-  ![](creator_to_cocos2dx-img/dialog_options.png "dialog options")
-
 * use the resulting dialog box to set the build options that you need.
```
Comment (drelaptop) [inline en/editors_and_tools/creator_to_cocos2dx.md]: when I try to use this plugin with cocos2d-x 3.16 and Creator 1.6.2, this step isn't here no longer. @minggo said this have been removed in new vision, should we remove this step in English docs?

Comment (slackmoehrle) [reply en/editors_and_tools/creator_to_cocos2dx.md]: What step? Just the picture? __Build__ is still required.

Comment (drelaptop) [reply en/editors_and_tools/creator_to_cocos2dx.md]: `* use the resulting dialog box to set the build options that you need.`
 this step `build options`

Comment (drelaptop) [reply en/editors_and_tools/creator_to_cocos2dx.md]: __build__ is required, __build options__ not

Comment (slackmoehrle) [reply en/editors_and_tools/creator_to_cocos2dx.md]: Done.

**Pull request merged**